assets/chezmoi.io/docs/user-guide/machines/windows.md

```diff
@@ -28,3 +28,23 @@ if (-Not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
   }
 }
 ```
+
+If you use [gsudo](https://gerardog.github.io/gsudo/docs/intro), their documentation specifies how to write self-elevating scripts [here](https://gerardog.github.io/gsudo/docs/tips/script-self-elevation#self-elevate-script).
+
+## Notes on running elevated scripts
+
+However you decide to get a script to run in an elevated prompt, as soon as the non-elevated script returns chezmoi will move to the next step in
+its processing (running more scripts, creating files, etc).
+So make sure that the elevated script actually completes *before the non-elevated one exits*, or subsequent scripts/steps might not run as expected
+if they depend on what the elevated script does.
+In the example above this is accomplished by passing the `-Wait` switch to PowerShell's `Start-Process` cmdlet.
+
+Also note that by including `-NoExit` as part of the parameters in `$CommandLine`, we're telling the new (elevated) PowerShell process/window to not exit
```

Comment:
This can be simplified, too:

```markdown
Note that by including `-NoExit` in `$CommandLine`, the new (elevated)
PowerShell process/window will not exit automatically on completion.
This means you'll need to close the new window by hand for chezmoi
to continue its steps.

If this manual intervention is desired, it would be convenient to have print
a message as its last command to indicate completion for you to safely
close the elevated window.

If you want no manual intervention, you can remove `-NoExit` from
`$CommandLine`, but then you likely won’t see the output of the elevated
script, which will make it more difficult to determine if something went wrong
during its execution.
```

Note that this latter will be helped when #2875 is merged, because then you
can test choose to include `-NoExit` based on whether `--verbose` is used
during `chezmoi apply`.